### PR TITLE
DOC: Misc typos fixed in docs and code comments

### PR DESCRIPTION
**Pull request by garanews**
Fix some typo

### CODE REVIEW

File: pandas/tests/io/sas/data/DRXFCD_G.csv [hunk 1]
```diff
@@ -4551,7 +4551,7 @@
 58101955,"TACO OR TOSTADA SALAD, MEATLESS W/ SOUR CREAM","Taco or tostada salad, meatless with sour cream"
 58103120,"TAMALE WITH MEAT","Tamale with meat"
 58103130,"TAMALE WITH CHICKEN","Tamale with chicken"
-58103200,"TAMALE, PLAIN, MEATLESS, NO SAUCE, PR STYLE","Tamale, plain, meatless, no sauce, Puerto Rican style or Carribean Style"
```
Comment (jbrockmendel) [inline pandas/tests/io/sas/data/DRXFCD_G.csv]: Is this causing the CI failure?  Id be OK with leaving this unchanged.

**Comment by jbrockmendel** [comment]
Small comment, generally looks good

File: pandas/tests/io/sas/data/DRXFCD_G.csv (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/io/sas/data/DRXFCD_G.csv]: It looks like yes, this is causing the failure.  Please revert.

File: pandas/tests/io/sas/data/DRXFCD_G.csv [hunk 2]
```diff
@@ -4551,7 +4551,7 @@
...
+58103200,"TAMALE, PLAIN, MEATLESS, NO SAUCE, PR STYLE","Tamale, plain, meatless, no sauce, Puerto Rican style or Caribbean Style"
```
Comment (datapythonista) [inline pandas/tests/io/sas/data/DRXFCD_G.csv]:
```suggestion
58103200,"TAMALE, PLAIN, MEATLESS, NO SAUCE, PR STYLE","Tamale, plain, meatless, no sauce, Puerto Rican style or Carribean Style"
```
I guess this is for tests with SAS, that need to march a SAS file with a typo. Let's revert here, and feel free to fix it in a separate PR for here, and for the SAS file.

**Comment by jbrockmendel** [comment]
@garanews can you apply the fix suggested by datapythonista?  this should be about ready otherwise

File: pandas/tests/io/sas/data/DRXFCD_G.csv (hunk 2 above)
Comment (garanews) [reply pandas/tests/io/sas/data/DRXFCD_G.csv]: let me know if is ok

**Comment by jbrockmendel** [comment]
lgtm cc @datapythonista

**Review by datapythonista — APPROVED** [review]

**Comment by datapythonista** [comment]
Thanks a lot for fixing all those @garanews 

If you want to open an issue for the typo in the csv that would be great. And also if you want to fix it yourself.

Btw, opening PRs from master is not a good practice, you'll have problems if you try to work in something else in parallel while the PR is being reviewed. Better to create a branch for every PR.

**Pull request merged**